test/versioned/ioredis/ioredis-3.tap.js

```diff
@@ -5,30 +5,32 @@
 
 'use strict'
 
-var tap = require('tap')
-var helper = require('../../lib/agent_helper')
-var assertMetrics = require('../../lib/metrics_helper').assertMetrics
-var params = require('../../lib/params')
-
-
-// CONSTANTS
-var DB_INDEX = 2
+const tap = require('tap')
+const helper = require('../../lib/agent_helper')
+const assertMetrics = require('../../lib/metrics_helper').assertMetrics
+const params = require('../../lib/params')
 
+// Indicates unique database in Redis. 0-15 supported.
+const DB_INDEX = 4
```

Comment:
This is the primary part for fixing the conflicts.

The rest of the code updates below are primarily to handle errors better for the tap framework.